aiida/cmdline/commands/cmd_status.py

```diff
@@ -95,21 +96,31 @@ def verdi_status(print_traceback, no_rmq):
         print_status(ServiceStatus.UP, 'repository', repository_status)
 
     # Getting the postgres status by trying to get a database cursor
-    database_data = [profile.database_name, profile.database_username, profile.database_hostname, profile.database_port]
+    backend_manager = manager.get_backend_manager()
+    dbgen = backend_manager.get_schema_generation_database()
+    dbver = backend_manager.get_schema_version_database()
+    database_data = [
+        profile.database_name, dbgen, dbver, profile.database_username, profile.database_hostname, profile.database_port
+    ]
     try:
         with override_log_level():  # temporarily suppress noisy logging
             backend = manager.get_backend()
             backend.cursor()
+
     except IncompatibleDatabaseSchema:
-        message = 'Database schema version is incompatible with the code: run `verdi database migrate`.'
+        message = f'Database schema version `Gen {dbgen} - v{dbver}` is incompatible with the code. '
```

Comment:
The notation `Gen {dbgen} - v{dbver}` will probably be confusing for SqlAlchemy databases. As they are hexadecimal stings, they can start with a letter and it won't be clear that the letter `v` is not part of it but the "version" marker. Maybe just have `Generation/version: {dbgen} / {dbver}`? Think the spaces are important so it is easy to select the version by double clicking

Reply:
Done. I didn't use exactly the template you put here but it is virtually the same, let me know what you think.

Reply:
I don't actually see the changes. Did you push it yet?

Reply:
Yes, that is weird. You don't see [these](https://github.com/aiidateam/aiida-core/pull/5228/files#diff-98113bf213b93ffbf8b7a306059cea0c28909c38179ef79c148814af298d449dR110-R127)? Github is buggy?